apps/simple-trading-app/src/app/components/simple-market-list/simple-market-list.tsx

```diff
@@ -32,6 +37,7 @@ export type RouterParams = Partial<{
 }>;
 
 const SimpleMarketList = () => {
+  const { isMobile } = useScreenDimensions();
```

Comment:
comment (non-blocking): imho this hook is overused in the app, mostly only to achieve visual adjustments easily doable from css-only, this instead triggers react to work on the reconciliation of the component tree, which i think is a lot less efficient. 

e.g.
```
{isMobile ? market.tradableInstrument.instrument.code : market.name}
```
->
```
<span className="display-inline md:hidden">{market.tradableInstrument.instrument.code}</span>
<span className="hidden md:display-inline">{market.name}</span>
```
etc.

Reply:
:+1:

Reply:
It's not only case of css classes, but also I use this `isMobile` boolean to give ag-grid widths of the columns, which are numbers, and are very useful for calculating its view dimensions.